net-misc/batctl/batctl-2020.2.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info toolchain-funcs
+
+DESCRIPTION="BATMAN advanced control and management tool"
+HOMEPAGE="https://www.open-mesh.org/"
+SRC_URI="https://downloads.open-mesh.org/batman/releases/batman-adv-${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
I noticed there was a LICENSES folder with 'deprecated' and 'preferred', and that the SPDX-License-Identifier header was used on files in the project.

Here's what turned up:
```
$ grep -rsin "SPDX-License-Identifier: MIT" .                                                                                                                                            
./batman_adv.h:1:/* SPDX-License-Identifier: MIT */
./LICENSES/preferred/MIT:7:    SPDX-License-Identifier: MIT
./list.h:1:/* SPDX-License-Identifier: MIT */
```

```
$ grep -rsin "SPDX-License-Identifier: ISC" .
./LICENSES/deprecated/ISC:7:    SPDX-License-Identifier: ISC
./genl.c:1:// SPDX-License-Identifier: ISC
```

Reply:
Added the two other licences.